packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -58,6 +68,27 @@ abstract class ProgressIndicator extends StatefulWidget {
   /// [ThemeData.accentColor].
   final Animation<Color> valueColor;
 
+  /// The [Semantics.label] for this progress indicator.
+  ///
+  /// This will be ignored if [semanticsEnabled] is false.
```

Comment:
This should include some information around how TalkBack/VoiceOver use this label to make an announcement.